### ISSUE TITLE
Formatter formats more than one file

### ISSUE THREAD

**Issue by RunDevelopment** [Galarius/vscode-opencl#15]
I noticed that when formatting one document, all other opened OpenCL documents were formatted as well. If I open two `.cl` files and format one of them, the other one will get formatted as well. If I open two `.cl` files, close the first file and format the second, both files will be formatted.

The most annoying part is that when you delete or rename a previously opened file, the formatter will log an error message:

```
no such file or directory
```

For every deleted or renamed file.

---

OpenCL extension setting:

<details>

```json
    "opencl.formatting.name": "\"C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\Llvm\\bin\\clang-format.exe\"",
    "opencl.formatting.options": [
        "-i"
    ],
```

</details>

**Comment by Galarius** [comment]
Can't reproduce it on macOS. Maybe you will get better results with `clang-format` that is shipped with `ms-vscode.cpptools` extension.

**Comment by RunDevelopment** [comment]
`ms-vscode.cpptools`' clang-format behaves the same. I tested this with and without setting the `-i` flag manually via `"opencl.formatting.options"`.

(I'm using VS's clang-format because `ms-vscode.cpptools` ships an older version that doesn't support a formatting option I use.)

**Comment by RunDevelopment** [comment]
Fixed in pre-release v0.6.5-pre.
